config/tutorials/tutorials.json

```diff
@@ -43,6 +43,12 @@
       "name": "Using Ably Client Library SDKs",
       "desc": "A selection of tutorials demonstrating the use of Realtime and/or REST libraries within devices such as browsers, desktops, mobiles or servers."
     },
+    "kafka": {
+      "id": "kafka",
+      "tab": "Channels, Push, Reactor & API Hub",
+      "name": "With Using Ably with Kafka",
```

Comment:
I think the first `With` is unnecessary.

Reply:
Well spotted, left over from experimentation! Thanks